code/controllers/subsystems/ghostroles.dm

```diff
@@ -105,15 +105,15 @@
 /datum/controller/subsystem/ghostroles/vueui_data_change(var/list/data, var/mob/user, var/datum/vueui/ui)
 	if(!data)
 		. = data = list("current_tag"="All")
-	LAZYINITLIST(data["spawners"])
+	data["spawners"] = list()
```

Comment:
Please do LAZYINITLIST, even if list isn't lazy, it just does what we need - if value is not set, set it to empty list. This breaks stuff as it ALWAYS sets list to empty list.